XTS/localjunit/WSTX11-interop/src/main/java/com/jboss/transaction/txinterop/interop/states/BAInteropUnsolicitedCompleteState.java

```diff
@@ -54,7 +49,7 @@ public BAInteropUnsolicitedCompleteState()
      */
     public ContentHandler getHandler(final ContentHandler nextHandler)
     {
-	return (replaced ? null : new RegistrationHandler(nextHandler)) ;
+	return new RegistrationHandler(nextHandler);
```

Comment:
It was there since the move from SVN. It is a test class, so my guess is, that it wasn't that important, since nobody ever came across it.